design_system/src/main/java/org/the_chance/honymart/ui/theme/Color.kt

```diff
@@ -15,6 +15,8 @@ val black16 = Color(0x29121212)
 val black37 = Color(0xFFA0A0A0)
 val black60 = Color(0xFFA7A7A7)
 val black87 = Color(0xDE121212)
+val owner_black60 = Color(0x99121212)
```

Comment:
Color naming in design system shouldn't depend on a specific app or feature, you can remove it and use the existing colors. 
You don't need to worry if they don't match Figma design. it will be changed with the design system cleanup.

Reply:
done